mocks/service/launchpad/index.js

```diff
@@ -44,10 +44,7 @@ router.get('/devel/*snaps', (req, res) => {
 });
 
 router.post('/~:owner/+snap/:name', (req, res) => {
```

Comment:
Good point.  I've fixed all the routes in this file (the need for the doubled backslash is documented in https://www.npmjs.com/package/path-to-regexp#custom-match-parameters).